app/apps/server/bridges/livechat.ts

```diff
@@ -166,10 +166,28 @@ export class AppLivechatBridge extends LivechatBridge {
 			type,
 		};
 
+		let userId;
+		let transferredTo;
+
+		if (targetAgent?.id) {
+			const targetUser = Users.findOneByAppId(targetAgent.id, {});
```

Comment:
Did you mean to use `fibdOneById` here instead?

Reply:
```suggestion
			const targetUser = Users.findOneById(targetAgent.id, { fields: { _id: 1, username: 1, name: 1 } });
```

Reply:
Hi, I made the change, pls check it again